example/Gemfile.lock

```diff
@@ -1,41 +1,49 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (3.0.5)
+    CFPropertyList (3.0.7)
```

Comment:
I think it's because React Native for iOS uses Ruby and CocoaPods  which is a tool for managing iOS dependencies  requires Ruby.

The major packages in `example/package.json` have been updated unless maybe you think there is any that I have missed.

Reply:
This is such a big PR, I thought `example/package.json` was just `package.json`. Looks good!